Sources/NIOHTTP1/NIOTypedHTTPServerUpgraderStateMachine.swift

```diff
@@ -393,5 +403,49 @@ struct NIOTypedHTTPServerUpgraderStateMachine<UpgradeResult> {
         }
     }
 
+    @usableFromInline
+    enum CloseInboundAction {
+        case close
+        case `continue`
+    }
+
+    @inlinable
+    mutating func closeInbound() -> CloseInboundAction {
```

Comment:
Naming NIT: Can we call this `inputClosed` instead please

Reply:
done